clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -559,7 +559,34 @@ describe("hubConnection", () => {
                 }
             });
 
-            if (transportType !== TransportType.LongPolling) {
+            it("can connect to hub with authorization using async token factory", async (done) => {
+                const message = "你好，世界！";
+
+                try {
+                    const hubConnection = new HubConnection("/authorizedhub", {
+                        accessTokenFactory: () => getJwtToken("http://" + document.location.host + "/generateJwtToken"),
```

Comment:
Is this from the JS async token PR? Curious why it is show up here. Did you move it?

Reply:
Because I split `Transports.ts` into four files (one for each transport and one for the interface) it got wiped out in a bad rebase.